modules/dnn/src/layers/convolution_layer.cpp

```diff
@@ -117,7 +117,7 @@ class BaseConvolutionLayerImpl : public ConvolutionLayer
         outputs_arr.getMatVector(outputs);
 
         CV_Assert((inputs.size() > outputs.size() && blobs.empty()) ||
-                  (!inputs.empty() && (blobs.size() == 1 || blobs.size() == 2)));
+                  (!inputs.empty() && blobs.size() >= 1));
```

Comment:
what this class should do with blobs.size() = 3 ?

This is assertion check about the code below.
It is caller responsibility to properly call this code without assertions